e2e/testdata/fn-eval/fn-source/exclude-meta-resources-by-default-out-of-place/.expected/exec.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+# create a temporary directory
```

Comment:
the test harness should create and delete the tmp dir automatically if bash scripts are provided. That eliminates lines 18,19,29,30. Can be a follow up.

Reply:
This temp dir is only used in out-of-place test cases because it needs a directory as another *place* to sink. I don't sink test harness should care about this kind of details.

Reply:
@droot has a PR for this #2163

Reply:
We discovered a bit late that using `.krmignore` and `special-dir` (`out` for out-of-place), these tests could be written differently where `exec.sh` scripts deleting/copying files around to get the `git diff` to right thing. If we have that pattern working well for our testing, we can potentially switch to that style in a follow up PR. I am fine having these as-is for now.